EIPS/eip-6051.md

```diff
@@ -0,0 +1,464 @@
+---
+eip: 6051
+title: Private Key Encapsulation
+description: defines a specification for encapsulating private keys.
+author: Base Labs (@Base-Labs), Weiji Guo (@weiji-cryptonatty)
+discussions-to: https://ethereum-magicians.org/t/private-key-encapsulation-to-move-around-securely-without-entering-seed/11604
+status: Draft
+type: Standards Track
+category: ERC
```

Comment:
This seems an `Interface` category

Reply:
I double checked eip-1 for `Interface` use. I don't think this should be an `Interface` category. The requests are optional for wallet applications. It does _not_ require _all_ wallet applications to implement. Also, EIP-5630 uses `ERC` category as well, while it also provides eth_ api additions like `eth_getEncryptionPublicKey ` and `eth_decrypt `.